src/controllers/__tests__/user.test.js

```diff
@@ -347,4 +358,178 @@ describe('User Endpoints', () => {
       expect(res.body[0]).toMatchObject(expectedDonatorResponse);
     });
   });
+
+  describe('POST /api/users/rating/:userid', () => {
```

Comment:
Correct all the route naming and API calls on the tests after restructuring the route path to `/:userid/rating`